.github/workflows/build+test.yml

```diff
@@ -35,3 +35,18 @@ jobs:
         with:
           name: coverage-report
           path: "${{github.workspace}}/app/build/reports/jacoco/unitTestCoverageReport/html/"
+      - name: Store generated release build APK for stagenet
+        uses: actions/upload-artifact@v3
+        with:
+          name: apk-stagenet-release
+          path: "${{github.workspace}}/app/build/outputs/apk/staging/release/"
+      - name: Store generated release build APK for testnet
+        uses: actions/upload-artifact@v3
+        with:
+          name: apk-testnet-release
+          path: "${{github.workspace}}/app/build/outputs/apk/prodTestNet/release/"
+      - name: Store generated release build APK for mainnet
+        uses: actions/upload-artifact@v3
+        with:
+          name: apk-mainnet-release
+          path: "${{github.workspace}}/app/build/outputs/apk/prodMainNet/release/"
```

Comment:
Since we are only building release versions we can simplify the paths
`path: "${{github.workspace}}/app/build/outputs/apk/prodTestNet/release/"`
can be
`path: "${{github.workspace}}/app/build/outputs/aab/prodTestNet/"`

Reply:
On my machine I also get the debug versions. Maybe it's because of the gradle task used in the workflow. Anyhow, if we skip the release part I believe the uploaded file would be in a `release` folder with I find undesirable. Will double check tomorrow.

Reply:
Confirmed: With the `release/` suffix the contents of the zip file are the apk and metadata files. Without it it has two folders `release` and `debug`, each with the apk and metadata files.